arch/arm/boot/dts/armada-385-ts7800-v2.dts

```diff
@@ -72,6 +72,25 @@
 		};
 	};
 
+	isa16550:  isa16550 { /* drivers/tty/serial/8250/ts7800_isa16550.c */
```

Comment:
the file `drivers/tty/serial/8250/ts7800_isa16550.c` doesn't exist, what is this comment hinting at?

Reply:
It means I forgot to bring that driver in.  I'll have to do that.